manifests/examples/azure/jessie.yml

```diff
@@ -23,7 +23,11 @@ volume:
     root:
       filesystem: ext4
       size: 10GiB
-packages: {}
+packages:
+  components:
+  - main
+  - contrib
+  - non-free
```

Comment:
When customers create VMs in cloud environments they often automate the deployment of software as well, so I'm trying to make that as easy as possible without the need to enable new repos or change configuration. I've found that sort of thing reduces friction, and more importantly customer tickets :)

I see this configuration in some of the other images as well.  But if you think it could cause issues I suppose I could just keep the default and make this change locally.